src/parquet/parquet_version.h

```diff
@@ -19,6 +19,6 @@
 #define PARQUET_VERSION_H
 
 // define the parquet created by version
-#define CREATED_BY_VERSION "parquet-cpp version 1.1.1-SNAPSHOT"
+#define CREATED_BY_VERSION "parquet-cpp version 1.2.1-SNAPSHOT"
 
-#endif  // PARQUET_VERSION_H
+#endif // PARQUET_VERSION_H
```

Comment:
Removing this file causes one of the Travis CI tests to fail. I am not sure why. I will look into this as well.